### PR TITLE
SCED Intro Tour, Part 5

### PR DESCRIPTION
**Pull request by Buhallin**
The penultimate step!  This completes the implementation for the tour itself, including:
- First draft tour script fully implemented
- Add investigator images as custom assets
- Added a field to skip re-centering the camera between cards
- Resized text area slightly to fill the space in the speech bubble

### CODE REVIEW

File: modsettings/CustomUIAssets.json [hunk 1]
```diff
@@ -113,5 +113,80 @@
...
+    "Type": 0,
+    "URL": "https://i.imgur.com/OgIv9N4.png"
+  },
+  {
+    "Name": "Inv-Gloria",
+    "Type": 0,
+    "URL": "https://i.imgur.com/xtYVdEL.png"
+  },
+  {
+    "Name": "Inv-Monterrey",
```
Comment (Chr1Z93) [inline modsettings/CustomUIAssets.json:158]: Typo in the name

File: src/core/tour/TourManager.ttslua [hunk 2]
```diff
@@ -110,10 +110,15 @@ do
   ---@param playerColor Player color to show the current card for
   internal.showCurrentCard = function(playerColor)
     internal.updateCardDisplay(playerColor)
-    local hook = getObjectFromGUID(tourState[playerColor].cameraHookGuid)
-    hook.setPositionSmooth(HOOK_CAMERA_HOME, false, false)
-    local delay = 0.5
+    local delay = 0
     local cardIndex = tourState[playerColor].currentCardIndex
+    local hook = getObjectFromGUID(tourState[playerColor].cameraHookGuid)
+--    local centerCamera = TOUR_SCRIPT[cardIndex].skipCentering and false or true
```
Comment (Chr1Z93) [inline src/core/tour/TourManager.ttslua:116]: Is this meant to stay as comment?

File: src/core/tour/TourManager.ttslua [hunk 3]
```diff
@@ -160,11 +165,12 @@ do
   ---@param playerColor Player color to update card for
```
Comment (Chr1Z93) [inline src/core/tour/TourManager.ttslua:165]: Nit: add variable type (String)

File: src/core/tour/TourManager.ttslua (hunk 2 above)
Comment (Buhallin) [reply src/core/tour/TourManager.ttslua:116]: Nope, should have been cleaned up.  Removed.

**Review by Chr1Z93 — APPROVED** [review]

**Pull request merged**